Description: Call the octave binaries by their name without version
Forwarded: not-needed
Author: Thomas Weber <tweber@debian.org>
--- a/scripts/pkg/private/configure_make.m
+++ b/scripts/pkg/private/configure_make.m
@@ -27,10 +27,9 @@ function configure_make (desc, packdir, verbose)
   if (exist (fullfile (packdir, "src"), "dir"))
     src = fullfile (packdir, "src");
     octave_bindir = octave_config_info ("bindir");
-    ver = version ();
-    mkoctfile_program = fullfile (octave_bindir, sprintf ("mkoctfile-%s", ver));
-    octave_config_program = fullfile (octave_bindir, sprintf ("octave-config-%s", ver));
-    octave_binary = fullfile (octave_bindir, sprintf ("octave-%s", ver));
+    mkoctfile_program = fullfile (octave_bindir, "mkoctfile");
+    octave_config_program = fullfile (octave_bindir, "octave-config");
+    octave_binary = fullfile (octave_bindir, "octave");
     cenv = {"MKOCTFILE"; mkoctfile_program;
             "OCTAVE_CONFIG"; octave_config_program;
             "OCTAVE"; octave_binary;
